frontend/packages/console-app/src/components/cloud-shell/setup/CloudShellSetup.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import { ALL_NAMESPACES_KEY } from '@console/shared';
+import { ProjectRequestModel } from '@console/internal/models';
+import { RootState } from '@console/internal/redux';
+import { connect } from 'react-redux';
+
+import { k8sCreate } from '@console/internal/module/k8s';
+import {
+  CloudShellSetupFormData,
+  CREATE_NAMESPACE_KEY,
+  cloudShellSetupValidation,
+} from './cloud-shell-setup-utils';
+import CloudSehellSetupForm from './CloudShellSetupForm';
+import { WorkspaceModel } from '../../../models';
+import { newCloudShellWorkSpace, createCloudShellResourceName } from '../cloud-shell-utils';
+
+interface StateProps {
+  activeNamespace: string;
+  username: string;
+}
+
+type Props = StateProps & {
+  onSubmit?: () => void;
+  onCancel?: () => void;
+};
+
+const CloudShellSetup: React.FunctionComponent<Props> = ({
+  activeNamespace,
+  onSubmit,
+  onCancel,
+  username,
+}) => {
+  const initialValues: CloudShellSetupFormData = {
+    namespace: activeNamespace === ALL_NAMESPACES_KEY ? undefined : activeNamespace,
+  };
+  const handleSubmit = async (values: CloudShellSetupFormData, actions) => {
+    actions.setStatus({ isSubmitting: true });
```

Comment:
```suggestion
    actions.isSubmitting(true);
```

Reply:
Interestingly enough it still showed up. I guess formik automatically sets submitting. Regardless I've updated the code.